lib/ClangImporter/ImportDecl.cpp

```diff
@@ -1535,7 +1535,7 @@ namespace {
         if (Name.str() == "id" && Impl.SwiftContext.LangOpts.EnableIdAsAny) {
           Impl.SpecialTypedefNames[Decl->getCanonicalDecl()] =
               MappedTypeNameKind::DoNothing;
-          return Impl.SwiftContext.getAnyDecl();
+          return nullptr;
```

Comment:
What do you want me to mark as unavailable, there is no result? My [initial implementation](https://github.com/apple/swift/pull/3293/commits/2612ea656f0a6de01caa7450f87814246676d961#diff-5c658ca134bff93fc4d435a34ecd5307R1543) constructed an implicit typealias for id, should I go back to doing that & mark it as unavaliable?